plugin/Installer/DatabaseChanges.php

```diff
@@ -241,6 +241,11 @@ public function install()
 			$dbversion = 31;
 		}
 
+		if ($dbversion == 31) {
```

Comment:
I merged master into this. Could you check that I handled this database migration correctly?

Reply:
Yes, that's right.